package/MDAnalysis/coordinates/DCD.py

```diff
@@ -130,6 +138,8 @@ def __init__(self, filename, convert_units=True, dt=None, **kwargs):
 
         .. versionchanged:: 0.17.0
            Changed to use libdcd.pyx library and removed the correl function
+        .. versionchanged:: 2.4.0
```

Comment:
I don't think that adding a deprecation warning requires a versionchanged. The deprecated in the class docs is sufficient.